lang/cc65/Portfile

```diff
@@ -12,43 +14,40 @@ long_description \
                 cc65 is a complete cross development package for 65(C)02 \
                 systems, including a powerful macro assembler, a C compiler, \
                 linker, librarian and several other tools.
+license         CC-BY-SA 
 
-# Per http://www.cc65.org/ discontinued support in 2013, moved to github
-# No version tags on github yet
 homepage        http://cc65.github.io/cc65
-distfiles       cc65-sources-${version}.tar.bz2
-use_bzip2       yes
+distfiles       V${version}.tar.gz
```

Comment:
No need for separate distfiles when using github.setup:
```suggestion
```

Reply:
done!